modules/core/src/lib/composite-layer.js

```diff
@@ -67,7 +67,8 @@ export default class CompositeLayer extends Layer {
       coordinateSystem,
       coordinateOrigin,
       wrapLongitude,
-      modelMatrix
+      modelMatrix,
+      subLayerProps: userProps
```

Comment:
Maybe change this function's parameter name so we don't need to alias like this? This function normally mainly takes id and updateTriggers?

Calling this `userProps` doesn't seem to make things clearer below. 

Maybe think about terms like "forwarding" and "override" to distinguish between the two processes going on here.